js/i18n-resources.js

```diff
@@ -136,6 +136,142 @@ var resources = { // English is fallback language.
+                "data source": 'De overlay gegevens komen van __OSMStartaTag__ en is bevraagd met __APIaTag__. De kaart is __OSMaTag__.',
+                "mode 0": 'Alleen tijdsperioden zijn toegestaan (zoals bij de tags opening_hours, lit)',
+                "mode 1": 'Individuele tijdspunten zijn toegestaan',
+                "mode 2": 'Zowel tijdsperioden als individuele tijdspunten zijn toegestaan (zoals bij de tags service_times, collection_times)',
+                "value to long for osm": 'De waarde is te lang voor OSM. De OpenStreetMap databank is momenteel beperkt tot waardes met __maxLength__ karakter. De mooi gemaakte waarde heeft een lengte van __pretLength__ en de ingevoerde waarde is __valLength__ karakters lang',
+                "low zoom level": 'De POIs verschijnen vanaf zoomniveau ${next}. U bent momenteel op zoomniveau ${actual}.',
+                "all n entries": 'Alle resultaten (__total__):',
+                "the first entries": 'De eerste __number__ van __total__ resultaten:',
+                "load all with JOSM": 'laad alles in JOSM',
+                "evaluation tool": 'evaluatie gereedschap',
+                "rule separator ;": 'scheidingsteken voor regel (de volgende regel zal een normale regel zijn)',
+                "rule separator  ||": 'scheidingsteken voor regel (de volgende regel will be a fallback rule which applies for any time not handled by previous rules)',
+                "rule separator ,": 'scheidingsteken voor regel (de volgende regel is aanvullende regel die de tijden van de vorige regels uitbreid. De volgende regel overschrijft de vorige regels dus niet (zoals een normale regel wel doet)',
+                "JOSM remote conn error": 'Kan niet connecteren met JOSM. Controleer aub dat  JOSM is opgestart en geconfigureerd is voor afstandbediening via de standaard tcp port 8111.',
+                "refer to yohours": 'Deze waarde kan ook geïnterpreteerd worden door YoHours. Dit is een eenvoudige editor voor de opening_hours syntax.  Indien je de geavanceerde mogelijkheden van de syntax niet nodig hebt, <a href=\"__href__\">probeer YoHours dan eens</a>.',
+            },
+            "words": {
+                "modifier": "__name__ wijziger (modifier)",
+                "selector": "__name__ keuzeschakelaar (selector)",
+                "mode": "evaluatie modus",
+                "green": "groen",
+                "yellow": "geel",
+                "red": "rood",
+                "violet": "violet",
+                "to": "tot",
+                "and": "en",
+                "no": "nee",
+                "his": "zijn",
+                "docu": "documentatie",
+                "of course": "natuurlijk",
+                "open": "open",
+                "unknown": "onbekend",
+                "closed": "gesloten",
+                "comment": "commentaar",
+                "today": "morgen",
```

Comment:
Fixed in https://github.com/opening-hours/opening_hours.js/commit/3e2c325fc583f643b9a83de831ee9a11d3097041